device/iot/manager.py

```diff
@@ -59,6 +58,14 @@ def __init__(self, state: State, recipe: RecipeManager) -> None:
         self.received_message_count = 0
         self.published_message_count = 0
 
+        # Initialize device info
+        self.device_id = registration.device_id()
+
+        # Initialize topics
+        self.config_topic = "/devices/{}/config".format(self.device_id)
```

Comment:
@jakerye I thought all these IoT and PubSub changes were already done in a previous PR?